multigeiger/display.cpp

```diff
@@ -91,20 +94,27 @@ static const char *status_chars[STATUS_MAX] = {
   // group TTN (LoRa WAN)
   ".t3T?",  // ST_TTN_OFF, ST_TTN_IDLE, ST_TTN_ERROR, ST_TTN_SENDING, ST_TTN_INIT
   // group BlueTooth
-  ".B4b?",  // ST_BT_OFF, ST_BT_CONNECTED, ST_BT_ERROR, ST_BT_CONNECTING, ST_BT_INIT
+  ".B4b?",  // ST_BT_OFF, ST_BT_CONNECTED, ST_BT_ERROR, ST_BT_CONNECTABLE, ST_BT_INIT
   // group other
   ".",      // ST_NODISPLAY
   ".",      // ST_NODISPLAY
   ".H7",    // ST_NODISPLAY, ST_HV_OK, ST_HV_ERROR
 };
 
 void set_status(int index, int value) {
-  if ((index >= 0) && (index < STATUS_MAX))
-    status[index] = value;
-  else
+  if ((index >= 0) && (index < STATUS_MAX)) {
+    if (status[index] != value) {
+      status[index] = value;
+      displayStatus();
+    }
+  } else
     log(ERROR, "invalid parameters: set_status(%d, %d)", index, value);
 }
 
+int get_status(int index) {
+  return status[index];
+}
```

Comment:
do you use that already?

Reply:
here, no. But in ble PR. As it is quite generic, I put it here together with the other display related commits.